Automattic/jetpack
# My Jetpack: add connection animation

## Proposed changes:

* This PR adds a new animation to the connection step on My Jetpack. This animation is intended to lessen user's confusion while Jetpack is connecting.


https://github.com/user-attachments/assets/ea806f49-d3e0-4f7a-81ae-8f61a2f70d68



### Other information:


## Jetpack product discussion



pbNhbs-bJV-p2

## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Spin up a JN site with Jetpack Beta pointing to this feature branch
* Go to My Jetpack
* Click to active your site with one click
* Confirm that the animation looks like the video above and there's no weird behavior
